src/persist-txn/src/txns.rs

```diff
@@ -829,30 +851,30 @@ mod tests {
         let mut txns = TxnsHandle::expect_open(client.clone()).await;
         let log = txns.new_log();
         let d0 = txns.expect_register(2).await;
-        txns.apply_le(&2).await;
```

Comment:
Oh yeah, good call. Did you audit all of these existing calls in the tests?

Reply:
Yeah, I removed any `apply_le(ts)` that followed an `expect_register(ts)`. Can you take a close look at `apply_and_tidy` though? I'm not 100% sure all the comments still make sense or are accurate.

Reply:
Yeah, I think `apply_and_tidy` comments still make sense to me. Left an unrelated thought I had when looking at it closely, feel free to ignore :)